modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -43,7 +43,11 @@ export default class NestedTabs extends Base {
 	}
 
 	getWidgetNumber() {
-		return this.$element.find( '> .elementor-widget-container > .e-n-tabs, > .e-n-tabs' ).attr( 'data-widget-number' );
+		const selector = elementorFrontend.config.experimentalFeatures.e_optimized_markup
```

Comment:
I am not sure why we need this, I had tried to cater for this change in the original code already.

Reply:
I have used to options, the one including the widget container, the one excluding the widget container.

I believe the DOM optimization should work out of the box here. At least, that's why I added it this way.

Reply:
@hein-obox Can you please elaborate? getWidgetNumber() will only work when .elementor-widget-container is present. How do you cover the scenario when it is not present?